### ISSUE TITLE
manual/Home.md

### ISSUE THREAD

**Issue by garbagetown** [playframework-ja/translation-project#859]
- https://github.com/playframework-ja/translation-project/tree/2.3.5/manual/Home.md
- diffs between [2.2.0/manual/Home.md](https://github.com/playframework-ja/translation-project/tree/2.2.0/manual/Home.md) and [2.3.5/manual/Home.md](https://github.com/playframework-ja/translation-project/tree/2.3.5/manual/Home.md).

```
--- /Users/garbagetown/Desktop/2.2.0/manual/Home.md 2013-09-19 10:11:22.000000000 +0900
+++ //Users/garbagetown/Desktop/2.3.5/manual/Home.md    2014-08-07 16:47:42.000000000 +0900
@@ -1,21 +1,22 @@
-# Play 2.2 documentation
+<!--- Copyright (C) 2009-2013 Typesafe Inc. <http://www.typesafe.com> -->
+# Play %PLAY_VERSION% documentation

 > Play is a high-productivity Java and Scala web application framework that integrates the components and APIs you need for modern web application development. 
 >
 > Play is based on a lightweight, stateless, web-friendly architecture and features predictable and minimal resource consumption (CPU, memory, threads) for highly-scalable applications thanks to its reactive model, based on Iteratee IO.

-- [[What's new in Play 2.2?|Highlights22]]
-- [[What's new in Play 2.1?|Highlights21]]
-- [[Play 2.2 Migration Guide|Migration22]]
+- [[What's new in Play 2.3?|Highlights23]] / [[Play 2.3 Migration Guide|Migration23]]
+- [[What's new in Play 2.2?|Highlights22]] / [[Play 2.2 Migration Guide|Migration22]]
+- [[What's new in Play 2.1?|Highlights21]] / [[Play 2.1 Migration Guide|Migration21]]

 ## Getting started

 1. [[Installing Play|Installing]]
 1. [[Creating a new application|NewApplication]]
 1. [[Anatomy of a Play application|Anatomy]]
-1. [[Using the Play console|PlayConsole ]]
+1. [[Using the Play console|PlayConsole]]
 1. [[Setting up your preferred IDE|IDE]]
-1. [[Sample applications|Samples]]
+1. [[Play Tutorials|Tutorials]]
 1. [Security policy](http://www.playframework.com/code/security)

 ## Working with Play
@@ -32,14 +33,18 @@
 1. [[Working with public assets|Assets]]
     1. [[Using CoffeeScript|AssetsCoffeeScript]]
     1. [[Using LESS CSS|AssetsLess]]
-    1. [[Using Google Closure Compiler|AssetsGoogleClosureCompiler]]
+    1. [[Using JSHint|AssetsJSHint]]
     1. [[Using RequireJS|RequireJS-support]]
 1. [[Working with the in-memory H2 database|Developing-with-the-H2-Database]]
 1. [[Managing database evolutions|Evolutions]]
 1. [[Configuration file syntax and features|Configuration]]
+    1. [[Configuring the application secret|ApplicationSecret]]
     1. [[Configuring the JDBC connection pool|SettingsJDBC]]
     1. [[Configuring Play's thread pools|ThreadPools]]
     1. [[Configuring logging|SettingsLogger]]
+    1. [[Configuring gzip encoding|GzipEncoding]]
+    1. [[Configuring security headers|SecurityHeaders]]
+    1. [[Configuring WS SSL|WsSSL]]
 1. [[Deploying your application|Production]]
     1. [[Creating a standalone package|ProductionDist]]
     1. [[Additional configuration|ProductionConfiguration]]
@@ -51,17 +56,20 @@

 1. [Scala](http://docs.scala-lang.org/)
 1. [Akka](http://akka.io/docs/)
-1. [sbt](http://www.scala-sbt.org/learn.html)
+1. [sbt](http://www.scala-sbt.org/0.13/tutorial/index.html)
 1. [Configuration](https://github.com/typesafehub/config)
 1. [Logback](http://logback.qos.ch/documentation.html)

 ## Hacking Play

 1. [[Building Play from source|BuildingFromSource]]
-1. [[3rd Party Tooling|ThirdPartyTools]]
+1. [[3rd Party Tools|ThirdPartyTools]]
 1. [[Repositories|Repositories]]
 1. [[Issue tracker|Issues]]
 1. [[Contributor guidelines|Guidelines]]
+1. [[Documentation guidelines|Documentation]]
+1. [[Translating Documentation|Translations]]
+1. [[Working with Git|WorkingWithGit]]

 ## About Play


```

**Comment by garbagetown** [comment]
やります :hand:

**Comment by garbagetown** [comment]
merged at #1046
